test/e2e/fixtures.go

```diff
@@ -474,3 +475,22 @@ func createTestBusyboxPods(tb testing.TB, data *TestData, num int, ns string, no
 
 	return podNames, podIPs, cleanupFn
 }
+
+// testServiceFromNode is a helper function which uses polling mechanism to make sure that
+// Service has been realized if created and Service is no longer accessible if deleted.
+func testServiceFromNode(t *testing.T, url, nodeName string, timeout time.Duration, statusReady bool) error {
```

Comment:
If testing from Node when proxyAll is disabled or kube-proxy is running, it's actually kube-proxy's rule serving the request. It's possible that antrea-agent haven't realized the service yet when it returns. So it's still not reliable.

Could we poll the flows/groups and check it eventually contains the required flows and groups within a timeout?

Reply:
Understood, i will work on the changes required.